auto-load-next-post.php

```diff
@@ -296,6 +296,8 @@ public function alnp_enqueue_scripts() {
 					'alnp_comments_container'   => get_option( 'auto_load_next_post_comments_container' ),
 					'alnp_remove_comments'      => get_option( 'auto_load_next_post_remove_comments' ),
 					'alnp_google_analytics'     => get_option( 'auto_load_next_post_google_analytics' ),
+					'alnp_event_on_load'        => get_option('auto_load_next_post_on_load_event'),
```

Comment:
@spatsatzis Space out to match other options. `get_option( 'auto_load_next_post_on_load_event' )`